src/Polytopes/Polyhedron/constructors.jl

```diff
@@ -60,16 +56,21 @@ pm_polytope(P::Polyhedron) = P.pm_polytope
 
 ### Construct polyhedron from V-data, as the convex hull of points, rays and lineality.
 @doc Markdown.doc"""
-    convex_hull(V [, R [, L]])
+    convex_hull(V::Matrix [, R::Matrix [, L::Matrix]]; non_redundant::Bool = false)
 
 The polytope given as the convex hull of the rows of a set of points.
 
-see Def. 2.11 and Def. 3.1  of [JT13](@cite)
+The matrices rows are the points, representative vectors of the rays and the generators of the lineality space,
+respectively.
+
+`R` can be given as an empty matrix or as `nothing` if the user wants to compute
+the convex hull only from `V` and `L`.
+
+If the user is sure that `V` and `R` only contains extreme points and that
+the description of the lineality space is complete, they can set
+`non_redundant = true` to avoid unneccessary redundancy checks.
 
-# Arguments
-- `V::Matrix`: Points whose convex hull is to be computed; encoded as row vectors.
-- `R::Matrix`: Rays completing the set of points; encoded row-wise as representative vectors.
-- `L::Matrix`: Generators of the Lineality space; encoded as row vectors.
```

Comment:
Ahh, OK. The information removed here is IMHO now missing above and needs to be added back to the running text somehow.